packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlDataFactoryInterface.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Shopsys\FrameworkBundle\Component\Router\FriendlyUrl;
 
+/**
+ * @method FriendlyUrlData createFromIdAndName(int $id, string $name)
```

Comment:
Please add a note about this into *US-6091 Resolving of deprecations for v8.0.0* in our internal backlog, I'm bit worried the method would not be added into the interface otherwise.

Reply:
I have already added a note there that it is necessary to resolve `@method` and `@internal` annotations, is it sufficient?

Reply:
Well this is the only `@method` that needs any kind of resolving, but fine